### PR TITLE
Pass exit code through for --exec option

### PR DESCRIPTION
**Pull request by mckornfield**

### CODE REVIEW

**Comment by mckornfield** [comment]
@benkehoe does this make sense to you, or would you prefer the exit code to not propagate as part of the `os.system` call?

File: aws_export_credentials/aws_export_credentials.py [hunk 1]
```diff
@@ -28,7 +28,6 @@
 import secrets
 
 from botocore.session import Session
-from botocore.credentials import ReadOnlyCredentials
```
Comment (mckornfield) [inline aws_export_credentials/aws_export_credentials.py:31]: looked unused

**Comment by benkehoe** [comment]
This seems like a good idea, I just haven't gotten a chance to deal with it yet

**Comment by benkehoe** [comment]
Can you make a `CONTRIBUTORS.md` in the root dir like this one and put us both on it? https://raw.githubusercontent.com/benkehoe/aws-sso-util/master/CONTRIBUTORS.md

**Comment by mckornfield** [comment]
Surely... @benkehoe addressed I believe

**Comment by benkehoe** [comment]
Shipped in v0.15

**Pull request merged**